AmericanMedicalAssociation/AMA-style-guide
# AMA-39: adds promo block directory, twig template with title, annotat…

…ion, and scss.

https://palantir.atlassian.net/browse/AMA-39

Adds the Promo block directory with twig template, annotation, and scss sheet. Twig template includes the title of the block. As the title is a h1 header, I do not think that an individual atom is necessary.

### Steps to Test:

1. Navigate to Molecules > Blocks > Promotional Block.
2. Observe that there are three Promotional Blocks with titles.
